src/MerchantCenter/AccountService.php

```diff
@@ -508,12 +508,44 @@ public function reset_wpcom_api_authorization(): bool {
 
 	/**
 	 * Update the status of the merchant granting access to Google's WPCOM app in the database.
+	 * Before update status in the DB it will comapre the nonce stored in the DB and the nonce passed to the API.
```

Comment:
Some typos here:

`Before updating the status in the DB it will compare the nonce stored in the DB with the nonce passed to the API.`